.gitignore

```diff
@@ -7,6 +7,12 @@ plugins-compat-tester-cli/dependency-reduced-pom.xml
 *.ipr
 *.iml
 
+### Eclipse
+.factorypath
+.classpath
+.settings/
+.project
+
```

Comment:
This is in a dedicated commit. This seemed small enough and not subject to create too much debate that I kept it here.

Reply:
fine with me